### DISCUSSION TITLE
Is it possible to filter multiple file types in ui.upload .props('accept=.cer|.json')

### DISCUSSION THREAD

**Discussion by ysharoiko** [zauberzeug/nicegui#2726] [Q&A]
Can we make possible to filter multiple file types in ui.upload 
Like .props('accept=.cer|.json')

**Comment by rodja** [comment]
Have a look at the Quasar documentation: https://quasar.dev/vue-components/uploader#uploading-multiple-files:~:text=Recommended%20format%20for,all%20file%20types. Looks like multiple options are simply space-separated.

**Reply by falkoschindler** [reply]
No, it's comma-separated. The documentation of the "accept" prop says: "Comma separated list of unique file type specifiers."

```py
.props('accept=".png, .jpg"')
```
